s3/replication/manager/src/s3replicationmanager/config.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
+
+#
+# Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+# Replication manager config
+#
+# This file contains config class and
+# functions to apply config
+
+# import sys
+import os
+# import shutil
+import logging
+import yaml
+# import uuid
+# import glob
+import logging.handlers
+
+class Config:
+    """ configuration for manager """
+
+    rep_conf = {}
```

Comment:
Please make these instance variables. 
https://docs.python.org/3/tutorial/classes.html#class-and-instance-variables

Reply:
done

Reply:
This change was not pushed by mistake. Taking care in next commit.